docs/rules/no-nested-tests.md

```diff
@@ -0,0 +1,38 @@
+# Ensure no tests are nested
+
+Translations: [Français](https://github.com/avajs/ava-docs/blob/master/fr_FR/related/eslint-plugin-ava/docs/rules/no-nested-tests.md)
```

Comment:
Yeah, I assumed that @forresst would work on it, as he has always been very quick on adding this. I just thought I'd make his job easier :sweat_smile:.